.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+# Description
+
+[Task name](link to task on the board goes here)
+
+Description of the task you completed and/or features added.
+
+## Acceptance Criteria
+
+Get the acceptance criteria for the story and/or task you are working on and put it all here.
+
+## Test Instructions
+
+1. Steps on...
+1. how to test...
+1. or inspect this...
+1. feature manually.
```

Comment:
This is probably a 1., 2., 3., 4. list, eh?

Reply:
You are correct, fixing now.

Reply:
Fixed